xilinx/arch.h

```diff
@@ -1544,23 +1545,21 @@ struct Arch : BaseArch<ArchRanges>
     bool isLogicTile(BelId bel) const
     {
         IdString belTileType = getBelTileType(bel);
-        return (belTileType == id_CLEL_L || belTileType == id_CLEL_R || belTileType == id_CLEM ||
-                belTileType == id_CLEM_R || belTileType == id_CLBLL_L || belTileType == id_CLBLL_R ||
-                belTileType == id_CLBLM_L || belTileType == id_CLBLM_R);
+        return (belTileType.in(id_CLEL_L, id_CLEL_R, id_CLEM, id_CLEM_R, id_CLBLL_L, id_CLBLL_R, id_CLBLM_L,
+                               id_CLBLM_R));
     }
     bool isBRAMTile(BelId bel) const
     {
         IdString belTileType = getBelTileType(bel);
-        return belTileType == id_BRAM || belTileType == id_BRAM_L || belTileType == id_BRAM_R;
+        return belTileType.in(id_BRAM, id_BRAM_L, id_BRAM_R);
     }
     /*bool isLogicTile(WireId wire) const
```

Comment:
this one can be remove, I left it commented, but it looks wrong since it is copy paste of one using BelId